week2/lab11/app.js

```diff
@@ -177,3 +186,40 @@ function resetPage(){
     images.push(img);
```

Comment:
you probably want to persist to local storage here since you just reset the images